### PR TITLE
fix: allow to send knock message when offline [AR-3162]

### PR DESCRIPTION
**Pull request by Garzas**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/AR-3162" title="AR-3162" target="_blank">BugAR-3162</a>  PlayTest 21.02 - Ping -> If sending a Ping fails, there should be a warning
  </td></table>
  <br />
 

----
#### PR Submission Checklist for internal contributors

- The **PR Title**
    - [x] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
    - [x] contains a reference JIRA issue number like `SQPIT-764`
    - [ ] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
    - [ ] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

### Issues

Sending pings was disabled when app is offline.

### Causes (Optional)

Sending ping should have similar behaviour as sending text messages and it should show error message under the sent message

### Solutions

Enable sending ping message while offline because rest of error handling is already implemented for system messages

### Attachments (Optional)

_Attachments like images, videos, etc. (drag and drop in the text box)_ 

| Before | After |
| ----------- | ------------ |
| before1 | after1  |

### CODE REVIEW

**Comment by AndroidBob** [comment]
Build [1](https://10.10.124.134/job/wire-android-reloaded/job/PR-1750/1/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv4.2.0(6406926)(6406926)-staging-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/staging/compat/com.wire.android-vv4.2.0(6406926)(6406926)-staging-compat.apk)

**Comment by AndroidBob** [comment]
Build [2](https://10.10.124.134/job/wire-android-reloaded/job/PR-1750/2/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv4.2.0(6413763)(6413763)-staging-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/staging/compat/com.wire.android-vv4.2.0(6413763)(6413763)-staging-compat.apk)

**Review by gongracr — APPROVED** [review]

**Comment by AndroidBob** [comment]
Build [3](https://10.10.124.134/job/wire-android-reloaded/job/PR-1750/3/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv4.2.0(6421546)(6421546)-staging-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/staging/compat/com.wire.android-vv4.2.0(6421546)(6421546)-staging-compat.apk)

File: app/src/main/kotlin/com/wire/android/ui/home/messagecomposer/MessageActionsBox.kt [hunk 1]
```diff
@@ -165,7 +165,7 @@ private fun AddMentionAction(isSelected: Boolean, addMentionAction: () -> Unit)
 private fun PingAction(onPingClicked: () -> Unit) {
     WireSecondaryIconButton(
         onButtonClicked = onPingClicked,
-        clickBlockParams = ClickBlockParams(blockWhenSyncing = true, blockWhenConnecting = true),
+        clickBlockParams = ClickBlockParams(blockWhenSyncing = false, blockWhenConnecting = false),
```
Comment (saleniuk) [inline app/src/main/kotlin/com/wire/android/ui/home/messagecomposer/MessageActionsBox.kt:168]: Sending and editing messages are now blocked when syncing, I don't see any reason now why it shouldn't work fine during syncing as we mainly had issues with "connecting" state, so when offline, but it's always better to make sure if that's fine, and if so maybe we could also enable sending and editing messages when syncing. 😄

**Review by saleniuk — APPROVED** [review]

**Comment by AndroidBob** [comment]
Build [4](https://10.10.124.134/job/wire-android-reloaded/job/PR-1750/4/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv4.2.0(6464620)(6464620)-staging-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/staging/compat/com.wire.android-vv4.2.0(6464620)(6464620)-staging-compat.apk)

**Pull request merged**